apps/docs/content/components/Bento.mdx

```diff
@@ -0,0 +1,227 @@
+| `verticalAlign`   | `"start"`, `"center"`, `"end"`, `ResponsiveAlign`                                                                          | `"start"`  | `false`  | The vertical alignment of children within the `Bento.Content`. This cannot be used with the `fixedBottomLink` property.                                   |
+| `fixedBottomLink` | `boolean`                                                                                                                  |  `false`   | `false`  | Sets the link within the `Bento.Content` to have a margin-top of `auto` pushing it to the bottom. With this set you can not vertically align the content. |
+
+The Bento.Content component API supports all standard HTML attribute props for the interface HTMLDivElement, while providing some additional behavior as described above.
+
+### Bento.Heading
+
+Required node that is used to provide a heading for the `Bento.Content`.
+
+| name        | type                       | default     | required | description                                           |
+| ----------- | -------------------------- | ----------- | -------- | ----------------------------------------------------- |
+| `className` | `string`                   |             | `false`  | Custom class name for the heading component           |
+| `children`  | `ReactNode`, `ReactNode[]` | `undefined` | `true`   | Content to be displayed inside the heading component. |
+
+Forwards all props from the [Heading component](/components/Heading), including `as`, `size`, and `weight`.
+
+### Bento.Visual
+
+| Name        | Type                                    |   Default   | Required | Description                                                                                              |
+| :---------- | :-------------------------------------- | :---------: | :------- | -------------------------------------------------------------------------------------------------------- |
+| `className` | `string`                                |             | `false`  | Sets a custom class on the root element                                                                  |
+| `fillMedia` | `boolean`                               |   `true`    | `false`  | Determines how the visual reacts within its container. Either filling the container or contained within. |
+| `position`  | `string`                                | `"50% 50%"` | `false`  | The positioning of the visual.                                                                           |
+| `padding`   | `"condensed"`, `"normal"`, `"spacious"` |             | `false`  | Sets a custom class on the root element                                                                  |
```

Comment:
The `padding` properties in Visual and Content are responsive right? if that's the case we might want to add the `ResponsivePaddingIndex` option here in both tables

Reply:
This has been updated in the following [SHA](https://github.com/primer/brand/pull/390/commits/c338ccd5c18e379757a65b3862d659a87bc9ba92)